app/code/Magento/InventoryLowQuantityNotification/Ui/Component/Product/Form/Element/UseConfigSettings.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryLowQuantityNotification\Ui\Component\Product\Form\Element;
+
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Data\ValueSourceInterface;
+use Magento\Framework\Serialize\JsonValidator;
+use Magento\Framework\Serialize\Serializer\Json;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Ui\Component\Form\Element\Checkbox;
+
+/**
+ * Class UseConfigSettings sets default value from configuration.
+ */
+class UseConfigSettings extends Checkbox
+{
+    /**
+     * @var Json
+     */
+    private $serializer;
+
+    /**
+     * @var JsonValidator
+     */
+    private $jsonValidator;
+
+    /**
+     * @param ContextInterface $context
+     * @param array $components
+     * @param array $data
+     * @param Json|null $serializer
+     * @param JsonValidator|null $jsonValidator
+     */
+    public function __construct(
+        ContextInterface $context,
+        $components = [],
+        array $data = [],
+        Json $serializer = null,
```

Comment:
It is new code
We don't need to follow BC

Reply:
Fixed.